app/models/payment.rb

```diff
@@ -0,0 +1,64 @@
+class Payment < ActiveRecord::Base
+  has_many :ticket_purchases
+  belongs_to :user
+  belongs_to :conference
+
+  attr_accessor :credit_card_number
+  attr_accessor :credit_card_type
+  attr_accessor :card_verification_value
+  attr_accessor :expiration_month
+  attr_accessor :expiration_year
+
+  validates :first_name, presence: true
```

Comment:
It's not being stored in the db.. at least there's no columns for this in the corresponding migration.